Update 'Create/Update an API key' (#1792)
elastic/docs-content: deploy-manage/api-keys/elasticsearch-api-keys.md

* **Cross-cluster** API key: allows other clusters to connect to this cluster.
* **Managed** API key: created and managed by {{kib}} to run background tasks.

> To manage API keys in {{kib}}, go to the **API Keys** management page using the navigation menu (**Management > Stack Management > API Keys**) or the [global search field](../../explore-analyze/find-and-organize/find-apps-and-objects.md).

Reviewer: I think parentheses can interrupt the flow of reading, so suggesting this instead:
```suggestion
To manage API keys in {{kib}}, go to **Management > Stack Management > API Keys** from the navigation menu or use the [global search field](../../explore-analyze/find-and-organize/find-apps-and-objects.md).
```

## Create an API key [create-api-key]

Two methods are available to create an API key:

* As a quick option to create a personal API key from anywhere in {{kib}}:

>   1. Select the **Help menu (![help icon](/deploy-manage/images/help-icon.svg

Reviewer: For consistency with other instructions on the page:
```suggestion
  1. From the **Help menu** (![help icon](/deploy-manage/images/help-icon.svg"")), select **Connection details > API key**.
```

>  "")) → Connection details → API key**.

Reviewer: ```suggestion

```

  1. Give the key a name.
  1. Select **Create API key**.
  
  Your personal API key is created with a default expiration of 90 days from the time of creation. You can manage the key from the **API Keys** page.

* To create a personal or cross-cluster API key with configurable options, go to the **API Keys** page using the navigation menu (**Management > Stack Management > API Keys**) or the [global search field](../../explore-analyze/find-and-organize/find-apps-and-objects.md), and select **Create API key**.

  ![Create API Key UI](/deploy-manage/images/kibana-create-user-api-key.png "")

  1. Choose to create either a user or a cross-cluster API key.
  2. Optionally, set an expiry date. By default the API key will not expire, but it's a good security practice to give the key a limited lifespan.
  3. Configure access:

>     * For a user API key, you can opt to configure access to specific {{es}} APIs and resources by assigning the key with predefined roles or custom privileges. Refer to [Defining roles](/deploy-manage/users-roles/cluster-or-deployment-auth/defining-roles.md) and the [Create API key](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-security-create-api-key) API documentation to learn more.

Reviewer: Could we indent these two bullets?
